Earthfile

```diff
@@ -611,6 +611,12 @@ ui-playwright-container:
     COPY web-console .
     COPY deploy/docker-compose.yml .
     COPY deploy/.env .
+
+    # Pull playwright-snapshots for visual regression testing
+    # It was decided it's better to clone the snapshots repo during the build rather than have it as a submodule
+    RUN echo https://github.com/feldera/playwright-snapshots.git$PLAYWRIGHT_SNAPSHOTS_COMMIT
+    RUN git clone --depth 1 https://github.com/feldera/playwright-snapshots.git$PLAYWRIGHT_SNAPSHOTS_COMMIT playwright-snapshots
```

Comment:
Consider using Earthly's `GIT CLONE` instead of `RUN git clone`: https://docs.earthly.dev/docs/guides/best-practices#git-clone-vs-run-git-clone